src/libsync/pushnotifications.h

```diff
@@ -101,19 +114,32 @@ private slots:
     void authenticateOnWebSocket();
     bool tryReconnectToWebSocket();
     void initReconnectTimer();
+    void pingWebSocketServer();
+    QByteArray timeoutPingPayload() const;
```

Comment:
No implementation for this

Reply:
argh. Why does the compile not throw a warning :(